ultravox/training/configs/llama_whisper.yaml

```diff
@@ -0,0 +1,103 @@
+# llama3.1-8b + whisper-medium, for development
+
+exp_name: "llama3.1-8b-whisper"
+text_model: "meta-llama/Meta-Llama-3.1-8B-Instruct"
+audio_model: "openai/whisper-medium"
+
+loss_config:
+  loss_function: "KL_Divergence"
+
+max_steps: 20 # x8x24 = 2,764,800
+
+# This would go in a datasets.yaml file and we could either use pyyaml-include to include it
+# or we could just add this logic to the training script. This file can also include its own datasets
+# key, with locally defined datasets.
+data_sets:
+  librispeech:
+    path: "fixie-ai/librispeech_asr"
+    user_template: "<|audio|>"
+    assistant_template: ""
+    transcript_template: "{{ text }}"
+
+  # Note the inheritance here
+  librispeech-clean:
+    base: "librispeech" # this could also be done via "<<": *librispeech, although that approach is less flexible
```

Comment:
Agreed. YAML variables are probably good for small-scale changes, but for this I like your approach more.